spec/requests/concerns/check_query_count_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe CheckQueryCount do
+  before do
+    allow(TradeTariffBackend).to receive(:check_query_count?).and_return true
+    allow(Raven).to receive(:capture_message).and_return true
+  end
+
+  let(:get_page) do
+    get api_commodity_path(commodity, format: :json)
+  end
+
+  let :commodity do
+    create :commodity,
+           :with_indent,
+           :with_chapter,
+           :with_heading,
+           :with_description,
+           :declarable
+  end
+
+  let(:alerter) { described_class::CountChecker.new(true) }
+
+  context 'with request count under threshold' do
+    before { get_page }
+
+    it 'does not raise an exception' do
```

Comment:
since we are not checking the exception in the test it sounds better to state:

`it 'returns success' do`

Reply:
in this scenario we're testing for the alerter behaviour so I'd say no, what we care about is whether the alerter triggered an exception